### PR TITLE
Ambassador langs

### PR DESCRIPTION
**Pull request by debs-obrien**
image

### CODE REVIEW

File: src/components/ProfileCard/index.tsx [hunk 1]
```diff
@@ -114,19 +122,23 @@ export function Ambassadors(): JSX.Element {
       ></ProfileCard>
       <ProfileCard
         name='John Hill'
+        languages="English"
```
Comment (mxschmitt) [inline src/components/ProfileCard/index.tsx:125]: why sometimes single and sometimes double quote?

Comment (debs-obrien) [reply src/components/ProfileCard/index.tsx:125]: cause you wont let me use prettier. I never ever look at single v double. Prettier has done that for me for years

Comment (mxschmitt) [reply src/components/ProfileCard/index.tsx]: Feel free to use whatever tools make you happy!

File: src/components/ProfileCard/styles.module.css [hunk 2]
```diff
@@ -38,28 +37,24 @@
...
+
 .socialLinks {
+  margin-top:.5rem;
   display: flex;
   justify-content: center;
-  padding-bottom: 2rem;
 }
 .socialLinks a{
-  padding: 16px;
+  margin-top: .5rem;
```
Comment (mxschmitt) [inline src/components/ProfileCard/styles.module.css:56]: why rem in this case and not pixels?

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**